src/locales/de/translation.json

```diff
@@ -57,7 +59,9 @@
   "Mitigation": "Linderung",
```

Comment:
Could you please address Richard's suggestion:
> I think a better translation of `Mitigation` in this context might be simply `Maßnahmen`.
https://github.com/neherlab/covid19_scenarios/pull/152#pullrequestreview-379309257
?

Reply:
Sure :+1: 
Sorry for late response